### DISCUSSION TITLE
Asking question about Uninstalling Streamlink Twitch GUI app (and it's empty folder)

### DISCUSSION THREAD

**Discussion by BoopFake7** [streamlink/streamlink-twitch-gui#794] [Q&A] [answered]
[Hi,](url) just recently I uninstalled Streamlink Twitch GUI app from my Windows 10 PC.
(by using it's Uninstall Streamlink Twitch GUI.exe file)

But before I started closing things down I notice that in my C:\Program Files folder, the Streamlink Twitch GUI folder is still there. But this time it's only has "assets" and "bin" within the folder.

So my main question is, can I just normally delete this slightly simi-empty Streamlink Twitch GUI folder?

I know this is a silly question to be ask, but I just wanted to make sure that all.

![Empty Streamlink Twitch GUI Folder 1](https://user-images.githubusercontent.com/82467938/114606629-66937f00-9c93-11eb-9032-1425299e1da7.PNG)
![Empty Streamlink Twitch GUI Folder 2](https://user-images.githubusercontent.com/82467938/114606637-6abf9c80-9c93-11eb-976c-12e1becdf38a.PNG)

**Comment by bastimeyer** [comment, marked as answer]
> can I just normally delete this slightly simi-empty Streamlink Twitch GUI folder?

Yes. All the installer does is extracting files from the embedded LZMA-archive into the target directory, creating a laucher shortcut and adding an entry to the system's registry for the uninstaller. The uninstaller has a list of all written files and then simply removes them and the entry from the registry when it gets executed.

> I know this is a silly question to be ask

No, it's alright. The uninstaller is supposed to keep track of all application files, but for some reason it doesn't remove the `bin` and `assets` subdirectories. These dirs were "added" a while ago when the app-content packaging method changed. Prior to that, the uninstaller removed everything correctly, but apparently something's wrong with the file list.

I think I've already found the cause of this. I'll push a fix later to the master branch, but until the next release, when uninstalling, just remove the left over files manually.

**Reply by bastimeyer** [reply]
Should be fixed by 00538678b14f47e3503243ce6727be9c04fd9bad.

**Reply by BoopFake7** [reply]
Hi bastimeyer, thanks for asking my question.
I have already delete them.

Thanks.
